src/basic/fileio.c

```diff
@@ -1327,33 +1327,31 @@ int read_timestamp_file(const char *fn, usec_t *ret) {
         return 0;
```

Comment:
extra points for turning the `->` into a proper unicode `→` in the commit msg ;-)